src/main/java/wooteco/subway/member/dao/MemberDao.java

```diff
@@ -39,7 +41,8 @@ public Member insert(Member member) {
 
     public void update(Member member) {
         String sql = "update MEMBER set email = ?, password = ?, age = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
+        jdbcTemplate.update(sql,
+            new Object[]{member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
```

Comment:
```suggestion
            member.getEmail(), member.getPassword(), member.getAge(), member.getId());
```
마지막 인자가 가변인자기 때문에, 배열은 제거해도 될 것 같아요 ^_^